examples/linux/okla-speedtest.yml

```diff
@@ -0,0 +1,17 @@
+integrations:
```

Comment:
This file already exists in a different path: https://github.com/newrelic/nri-flex/blob/master/examples/okla-speedtest.yml

Could you move it to the desired path? `examples/linux/oka-speedtest.yml`?

Reply:
Done.